### PR TITLE
testcases 2

### PR DESCRIPTION
**Pull request by TheSharpOwl**

### CODE REVIEW

File: examples/10.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    y := 10
+end
+
+routine ifFive (x) : boolean is
+    return x = 5
+end
+
+type bool is boolean
+
+routine isSix (z) : bool is
```
Comment (matveyplevako) [inline examples/10.txt]: I think you've missed type for argument

File: examples/10.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if z = 6 then
+        return true
+    else
+        return
```
Comment (matveyplevako) [inline examples/10.txt]: missing false here

File: examples/10.txt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    end
+end
+
+var z is 5 + x * y / (4 = 4)
+
+var a : array[10] bool
+a[10] := 0
+a[9] = true
```
Comment (matveyplevako) [inline examples/10.txt]: should be := instead of =

File: examples/10.txt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+a[8] = 9.8 \= 50
```
Comment (matveyplevako) [inline examples/10.txt]: same

File: examples/11.txt [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            if b = (a[j] < a[i]) then
+                var z is a[i]
+                a[i] := a[j]
+                a[j] := z
+            end
+        end
+    end
+end
+
+var a is array [10] integers
```
Comment (matveyplevako) [inline examples/11.txt]: replace here **is** with **:**

File: examples/10.txt [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    x := true
+end
+
+if x then
+    x := 5.3
+else
+    y := 10
+end
+
+routine ifFive (x) : boolean is
```
Comment (matveyplevako) [inline examples/10.txt]: I think you've missed type for argument

File: examples/11.txt [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+routine fill(a:array [] integer, x : integer) is
+    for i in range 1 .. a.size loop
+        a[i] := x
+    end
+end
+
+
+# this is a comment (you can delete it later)
+# sorting function, if b is true, it makes a in non decreasing order
+# otherwise it will be in increasing order
```
Comment (matveyplevako) [inline examples/11.txt]: Can you delete them, please? as we didnt have syntax for them in doc

File: examples/11.txt [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+routine sort(a : array [] integer, b : boolean) is
+    for i in range 1 .. a.size loop
+        for j in range i .. a.size loop
```
Comment (matveyplevako) [inline examples/11.txt]: we don't have range keyword, just ..

File: examples/12.txt [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+routine get5th(a : array [] integer) : integer is
+    if a.size < 5 then
+        return -1
+    else
+        return a[5]
+    end
+end
+
+type ninja as record
```
Comment (matveyplevako) [inline examples/12.txt]: replace as with is

File: examples/12.txt [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    var HP : integer
+    var color : bool
+    var inventory : array [100] integer
+end
+
+routine doDamage(nj : ninja, damage : integer) is
+    nj.HP = nj.HP - damage
```
Comment (matveyplevako) [inline examples/12.txt]: should be :=

File: examples/9.txt [hunk 11]
```diff
@@ -0,0 +1,28 @@
+var x : integer is 5 + 5
+var y is 5.10 
+
+routine add5(z) : integer is
```
Comment (matveyplevako) [inline examples/9.txt]: add argument type here

File: examples/9.txt [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+    return z + 5
+end
+
+
+routine go5times() is
+    for x in reverse 5..1 loop
+    end
+    for x in 1..5 loop
+    end
+
```
Comment (matveyplevako) [inline examples/9.txt:15]: missing end for routine go5times()

File: examples/9.txt [hunk 13]
```diff
@@ -0,0 +1,29 @@
+var x : integer is 5 + 5
+var y is 5.10 
+
+routine add5(z : integer) : integer is
+    return z + 5
+end
+
+
+routine go5times() is
```
Comment (pfh-hbq) [inline examples/9.txt]: add an argument to the routine

**Pull request merged**